azure-pipelines.yml

```
@@ -172,37 +166,7 @@ jobs:
       git checkout "$(RISCV_COMPLIANCE_GIT_VERSION)"
     displayName: Get RISC-V Compliance test suite
 
-  - bash: |
-      # Build simulation model of Ibex
-      fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ibex:ibex_riscv_compliance --RV32M=1 --RV32E=0
-      if [ $? != 0 ]; then
-        echo -n "##vso[task.logissue type=error]"
-        echo "Unable to build Verilator model of Ibex for compliance testing."
-        exit 1
-      fi
-
-      # Run compliance test suite
-      export TARGET_SIM=$PWD/build/lowrisc_ibex_ibex_riscv_compliance_0.1/sim-verilator/Vibex_riscv_compliance
-      export RISCV_PREFIX=riscv32-unknown-elf-
-      export RISCV_TARGET=ibex
-      export RISCV_DEVICE=rv32imc
-      fail=0
-      for isa in rv32i rv32im rv32imc rv32Zicsr rv32Zifencei; do
-        make -C build/riscv-compliance RISCV_ISA=$isa 2>&1 | tee run.log
-        if [ ${PIPESTATUS[0]} != 0 ]; then
-          echo -n "##vso[task.logissue type=error]"
-          echo "The RISC-V compliance test suite failed for $isa"
-
-          # There's no easy way to get the test results in machine-readable
-          # form to properly exclude known-failing tests. Going with an
-          # approximate solution for now.
-          if [ $isa == rv32i ] && grep -q 'FAIL: 4/48' run.log; then
-            echo -n "##vso[task.logissue type=error]"
-            echo "Expected failure for rv32i, see lowrisc/ibex#100 more more information."
-          else
-            fail=1
-          fi
-        fi
-      done
-      exit $fail
-    displayName: "Run RISC-V Compliance test for Ibex RV32IMC"
+  # Run Ibex RTL CI per supported configuration
+  - template : ci/ibex-rtl-ci-steps.yml
+    parameters:
+      ibex_configs : ['small-fastmult', 'maxperf-scmult']
```

It's easier to extend these lists (and produce smaller diffs) if you go for the "longer list style":

```suggestion
      ibex_configs: 
      - small-fastmult
      - maxperf-scmult
```